services/__tests__/date.test.tsx

```diff
@@ -0,0 +1,11 @@
+import { getDate, getDateTime } from "../date";
+
+it("it extracts the date from an ISO string and equals to: 2 dec. 2011", () => {
```

Comment:
Usually convention for writing test names is to have a consistent and clearly described test. I do think writing `it("it ...` should be a crime though 🤷‍♂️  You can do this though

```suggestion
test("it extracts the date from an ISO string and equals to: 2 dec. 2011", () => {
```